api/models/v0.3/modelPolicy.js

```diff
@@ -1,67 +1,40 @@
 const IlsClient = require("../../controllers/v0.3/IlsClient");
 
-const lowerCase = (arr) => arr.map((item) => item.toLowerCase());
-
 /**
  * Creates a policy object to find out what type of card is allowed for a
- * given patron and their location.
+ * given card and their location.
```

Comment:
Why the change from "patron" to "card" here?

Reply:
Because the class is called `Card`. I didn't make updates elsewhere but I can fix that naming issue in a later PR.

Reply:
I did make that mistake from the Ruby Card Creator where it used Patron and Card interchangeably and it would be better to just use `Card`.